novawallet/Modules/Vote/Governance/ReferendumVoteConfirm/Presenter/BaseReferendumVoteConfirmPresenter.swift

```diff
@@ -177,13 +177,13 @@ class BaseReferendumVoteConfirmPresenter {
         fatalError("Must be overriden by subsclass")
     }
 
-    func didReceiveVotingReferendum(_: ReferendumLocal) {
-        fatalError("Must be overriden by subsclass")
-    }
+    func didReceiveVotingReferendumsState(_ state: ReferendumsState) {
+        votesResult = state.voting
 
-    func didReceiveVotingHash(_: String) {
-        fatalError("Must be overriden by subsclass")
+        refreshLockDiff()
     }
+
+    func didReceiveVotingHash(_: String) {}
```

Comment:
Also this one is used only needed for ReferendumConfirmInteractor. So, I would move there